### ISSUE TITLE
NVR's IP has no IP chanNum

### ISSUE THREAD

**Issue by BilalS10** [YOHO-LAB/save-hikvision-record#2]
I used the library and successfully compiled it as well.
But I got the above error while working through it.

complete log is : 

======= success get nvr info========UserID: 0
DevType: 31
IPcamNum: 0 
StartDChan: 0 
HighDChanNum: 0 
SerialNumber: DS-2CD5A26G0-IZS20191125AAWRD91919905
NVR's IP has no IP chanNum

how to solve this problem?

**Comment by andybrier** [comment]
Please make sure that the NVR is IP Camara.

**Comment by BilalS10** [comment]
I removed this check and got it working. But now im facing another issue which is after providing it the start and end time still it isnt able to download full video for that time. The videos are most of the time incomplete ( ie: if i have given 3 minutes it will download for 2 minutes only or even less.) I checked in the playback, the footage is there. Kindly help me with the same.

Thank you

**Comment by BilalS10** [comment]
Yes it is an ip camera. This issue got resolved by commenting this check and direclty starting the video download
